[2025-07-22 18:31:03][neurapy_logger.drawer][INFO] : Logging to file: Application_Real_Time\Robot\data\logs\drawing_drawer_2025-07-22_18-31-03.log
[2025-07-22 18:31:03][neurapy_logger.drawer][INFO] : Logging to file: Application_Real_Time\Robot\data\logs\drawing_drawer_2025-07-22_18-31-03.log
[2025-07-22 18:31:03][neurapy_logger.drawer][INFO] : Logging to file: Application_Real_Time\Robot\data\logs\drawing_drawer_2025-07-22_18-31-03.log
[2025-07-22 18:31:03][neurapy_logger.drawer][INFO] : Robot initialized and ready.
[2025-07-22 18:31:03][neurapy_logger.drawer][INFO] : Robot initialized and ready.
[2025-07-22 18:33:24][neurapy_logger.drawer][ERROR] : Unhandled exception: No objects to concatenate
[2025-07-22 18:33:24][neurapy_logger.drawer][ERROR] : Unhandled exception: No objects to concatenate
[2025-07-22 18:33:47][neurapy_logger.drawer][ERROR] : Unhandled exception: 
[2025-07-22 18:33:47][neurapy_logger.drawer][ERROR] : Unhandled exception: 
[2025-07-22 18:33:48][neurapy_logger.drawer][ERROR] : Unhandled exception: 
[2025-07-22 18:33:48][neurapy_logger.drawer][ERROR] : Unhandled exception: 
[2025-07-22 18:33:48][neurapy_logger.drawer][ERROR] : Unhandled exception: 
[2025-07-22 18:33:48][neurapy_logger.drawer][ERROR] : Unhandled exception: 
[2025-07-22 18:33:48][neurapy_logger.drawer][ERROR] : Unhandled exception: 
[2025-07-22 18:33:48][neurapy_logger.drawer][ERROR] : Unhandled exception: 
